homeassistant/components/concord232/alarm_control_panel.py

```diff
@@ -69,7 +70,7 @@ def __init__(self, url, name, code, mode):
         self._url = url
         self._alarm = concord232_client.Client(self._url)
         self._alarm.partitions = self._alarm.list_partitions()
-        self._alarm.last_partition_update = datetime.datetime.now()
+        self._alarm.last_partition_update = dt_util.utcnow()
```

Comment:
What effect will this have on the alarm?

Reply:
I couldn't find any reference to this variable in the concord232 repo:
https://github.com/JasonCarter80/concord232

Also, I couldn't find any use in our repo. I think we could actually delete this line. @JasonCarter80  would you mind explaining this line?

Reply:
I'm in favor of deleting it if there's no use here or in the library. I can't see anything either.

Reply:
Deleted